airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SimpleLeftSide/SimpleLeftSide.module.scss

```diff
@@ -0,0 +1,15 @@
+@use "scss/colors";
+@use "scss/variables";
+
+.checkIcon {
+  color: colors.$dark-blue-100;
+}
+
+.detailTextContainer {
+  min-width: fit-content;
+  color: colors.$dark-blue;
+}
+
+.detailsContainer {
+  overflow: visible;
+}
```

Comment:
Trying to understand why we need this? I tried to remove it and I can't see any difference (no matter the width of the window). Could you clarify what use-case this is for?

Reply:
Sure! Details were being cutted off, but it seems that is working now, sorry!